src/test/blockchains.test.js

```diff
@@ -570,4 +574,105 @@ describe('Blockchains route', async function () {
       field: 'blockchainId',
     });
   });
+  describe('POST useful address', () => {
+    describe('Core of the route', () => {
+      it('Should return 403 if no token is not provided', async function () {
+        const res = await chai
+          .request(app)
+          .post('/test/blockchains/useful-address/1234');
+        chai.expect(res).to.have.status(403);
+      });
+      it('Should create a new useful address with the proper fields', async function () {
```

Comment:
@juniormp can you test also the situation where `myContract1` already exists and so you are modifying the address with this call?